### ISSUE TITLE
aspect-creation does not work with accessibility Features enabled

### ISSUE THREAD

**Issue by joshjama** [gsantner/dandelion#288]
Aspect creating is not possible with accessibility-features enabled. 
As a blind Person i'm using screenreader software to read the text out loud to me. If I click on create new aspect nothing happens on mobile sites. On the  Desktop-Version the site crashes after then. 

#### General information

* **App version:** 
* **System:** 
* **Pod:**

**Comment by gsantner** [comment]
does it work when you login at the pod in normal web browser?

**Comment by joshjama** [comment]
Hi, it does not work in normal web browser. 
Don't I have to be signed in to create an aspect which is related to me? 
How to Login anonym? 
Best 
Joshua
Am 25.04.2021 12:13 schrieb Gregor Santner ***@***.***>:
does it work when you login at the pod in normal web browser?

**Comment by gsantner** [comment]
You enter the pod url in your web browser on your phone. Yes login, enter username/password. Of course you need to be logged in to view **your** aspects.

dandelion is just the website of your pod, the diaspora web frontend.

**Comment by joshjama** [comment]
That may be the case, but my problem is also a frontend Problem.
Am 25.04.2021 12:47 schrieb Gregor Santner ***@***.***>:
You enter the pod url in your web browser on your phone, then enter username/password.
dandelion is just the website of your pod, the diaspora web frontend.

**Comment by gsantner** [comment]
If it also happens in web browser I cannot help you, the issue is in diaspora project then, not at dandelion.

dandelion just displays diasporas mobile web appearance.
